python/lvmdrp/functions/run_drp.py

```diff
@@ -956,6 +961,110 @@ def combine_cameras(tileid: int, mjd: int, spec: int = 1):
         log.info(f'Output combined camera file: {bout_file}')
 
 
+def combine_spectrographs(tileid: int, mjd: int, expnum: int) -> fits.HDUList:
```

Comment:
I see you added the option to spec-combine one exposure, can we get the same for `combine_cameras`? I can see users wanting to channel-combine only one exposure.

Reply:
I'm not sure it's really necessary.  In general, users won't be running these functions, which is true for the majority of the functions in the DRP.  They only care about the final data products.   In any case, that's a separate feature request outside the scope of this PR.   If you really want that option, I can add it in a separate work item.

Reply:
Yes, I agree this should be done in a separated feature and it's not priority right now. I do think it is a good option to have though.